subsys/net/l2/wifi/wifi_shell.c

```diff
@@ -101,6 +110,87 @@ static void handle_wifi_scan_result(struct net_mgmt_event_callback *cb)
 					     sizeof(mac_string_buf)) : ""));
 }
 
+#ifdef CONFIG_WIFI_FEAT_RAW_SCAN_RESULTS
+static int wifi_freq_to_channel(int frequency)
+{
+	int channel = 0;
+
+	if (frequency == 2484) { /* channel 14 */
+		channel = 14;
+	} else if ((frequency <= 2472) && (frequency >= 2412)) {
+		channel = ((frequency - 2412) / 5 )+ 1;
+	} else if ((frequency <= 5320) && (frequency >= 5180)) {
+		channel = ((frequency - 5180) / 5 )+ 36;
+	} else if ((frequency <= 5720) && (frequency >= 5500)) {
+		channel = ((frequency - 5500) / 5 )+ 100;
+	} else if ((frequency <= 5895) && (frequency >= 5745)) {
+		channel = ((frequency - 5745) / 5 )+ 149;
+	} else {
+		channel = frequency;
+	}
+
+	return channel;
+}
+
+static enum wifi_frequency_bands wifi_frequency_to_band(int frequency)
```

Comment:
Please be consistent with the naming - either use `wifi_freq_`to_* in both cases or `wifi_frequency_to*`.